src/pages/Fox/components/FoxHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import { Container, Flex, Heading, Stack } from '@chakra-ui/react'
+import { fromAccountId } from '@shapeshiftoss/caip'
+import { useCallback, useEffect, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { AccountDropdown } from 'components/AccountDropdown/AccountDropdown'
+import { Display } from 'components/Display'
+import { InlineCopyButton } from 'components/InlineCopyButton'
+import { PageBackButton, PageHeader } from 'components/Layout/Header/PageHeader'
+import { Text } from 'components/Text'
+import { selectPortfolioAccountIdsByAssetIdFilter } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { useFoxPageContext } from '../hooks/useFoxPageContext'
+
+const containerPadding = { base: 6, '2xl': 8 }
+const containerPaddingTop = { base: 0, md: 8 }
+const accountDropdownButtonProps = { variant: 'solid', width: 'full' }
+
+export const FoxHeader = () => {
+  const translate = useTranslate()
+  const { assetId, setAssetAccountId, selectedAssetAccountId } = useFoxPageContext()
+  const accountIdsFilter = useMemo(() => ({ assetId }), [assetId])
+  const accountIds = useAppSelector(state =>
+    selectPortfolioAccountIdsByAssetIdFilter(state, accountIdsFilter),
+  )
```

Comment:
blocking: this should narrow down by accountNumber, honouring this guy 
<img width="265" alt="image" src="https://github.com/user-attachments/assets/1509caf1-66a7-4da2-be94-2887e0db28b7">

Reply:
Actually disregard me, we need this for the dropdown, but we should be able to ditch the effect below, see inline comment